packages/orchestration/src/examples/README.md

```diff
@@ -0,0 +1,11 @@
+# Examples
+
+This directory contains sample contracts showcasing the Orchestration API. Each example demonstrates specific functionalities:
+
+- **stakeIca.contract.js**: Interchain account creation for remote staking
+- **basic-flows.contract.js**: Account creation and query sending
+- **unbondExample.contract.js**: Cross-chain unbonding and liquid staking
+- **sendAnywhere.contract.js**: Token sending across supported blockchains
+- **swapExample.contract.js**: Token swapping and remote staking
```

Comment:
This one doesn't really work yet either. `makeOsmosisSwap` isn't completely implemented.

 - #8863